addons/mail/models/ir_translation.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _
+from odoo.exceptions import AccessError
+
+
+class IrTranslation(models.Model):
+    _inherit = 'ir.translation'
+
+
+    def create(self, vals_list):
+        res = super().create(vals_list)
+        res._check_is_dynamic()
+        return res
+
+    def write(self, vals):
+        res = super().write(vals)
+        self._check_is_dynamic()
+        return res
+
+    def _check_is_dynamic(self):
+        # if we don't modify translation of at least a model that inherits from mail.render.mixin, we ignore it
+        translations_for_mail_render_mixin = self.filtered(lambda tmpl: isinstance(self.env[tmpl.name.split(',')[0]], self.pool['mail.render.mixin']))
```

Comment:
instead of isinstance shouldn't we use subclass ?

Reply:
both works, but yes, I can change it, more readable :+1: 
and more coherent with the code of `_reflect_model_params`